### ISSUE TITLE
Plugin not activated, looking for dev although deploy arg is --stage local 

### ISSUE THREAD

**Issue by thetumper** [localstack/serverless-localstack#50]
Receiving this message during startup:

Serverless: serverless-localstack plugin not activated. 'dev' is not present in config custom.localstack.stages

It should not be defaulting to dev, as --stage local is sent as an arg to sls deploy, and  configuration should activate only for local:

From serverless.yml:
```
# Service Name
service: ${file(package.json):name}

# Serverless framework version
# -----------------------------
frameworkVersion: '>=1.26.0 <2.0.0'

# Provider Configurations
# ------------------------
provider:
  name: aws
  runtime: nodejs8.10
  region: ${opt:region, 'us-east-1'}

  # Application Environment
  stage: ${opt:stage, 'local'}

  # Configure the S3 bucket where your application package is stored
  deploymentBucket:
    name: ${file(serverless/stages/${self:provider.stage}.yml):sls.deploymentBucket}
    serverSideEncryption: AES256

  # Disable lambda versioning
  versionFunctions: false

  # Enable X-Ray tracing (serverless-plugin-tracing)
  tracing: true

  # IAM Role used for Deployment
  #   This is always account-specific and is different for each environment
  cfnRole: ${file(serverless/stages/${self:provider.stage}.yml):sls.cfnRole}

  # IAM Role used by Lambda
  #   This is a resource defined in serverless/cloudformation-resources.yml
  role: LambdaDefaultIamRole

  # Memory Configurations
  #   Max is ~3GB. Also provides 2 CPUs
  memorySize: 3008

  # Set the timeout to 28 seconds. Max allowed is 30
  timeout: 28

  # Application Environment Variables
  environment:
    APP_NAME: ${self:service}
    APP_VERSION: ${file(package.json):version}
    APP_ENV: ${self:provider.stage}

plugins:
  - serverless-plugin-aws-alerts
  - serverless-plugin-tracing
  - serverless-pseudo-parameters
  - serverless-webpack
  - serverless-localstack

custom:
  # Configurations for serverless-plugin-aws-alerts
  alerts: ${file(serverless/lambda-cloudwatch-alerts.yml)}

  # Configurations for serverless-webpack
  webpack:
    includeModules: true

  # Configuration for serverless-localstack
  localstack:
    debug: true
    stages: ['local']
    autostart: false

# Function Configurations
# -------------------------
functions:
  myFunction:
    handler: src/lambda.handler
    description: ${file(package.json):description}
    events: ${file(serverless/stages/lambda-events.${self:provider.stage}.yml):myEvents}

# CloudFormation Resources
# -------------------------
resources:
  - ${file(serverless/cloudformation-resources.yml)}
```

**Comment by whummer** [comment]
Thanks for reporting @thetumper . What is the exact command you're running? It *should* work using `sls deploy --stage local`. Having troubles reproducing this issue - do you think you can provide a small, self-contained test example that does not depend on external resources? Thanks!

**Comment by thetumper** [comment]
Thanks for checking.  Yes, I will try to do that soon.

**Comment by tmcgaughey** [comment]
I found something similar.  This plugin only appears to execute if the matching stage is specified on the command line e.g.: "--stage local", but most plugins will work fine with the "provider: stage" variable set.

**Comment by whummer** [comment]
Thanks @tmcgaughey @thetumper  . Can you please give it a try with the latest version (0.4.20) and see if the problem persists?

Your example seems to be working for me with the following config:
```
$ sls --version
Framework Core: 1.51.0
Plugin: 1.3.10
SDK: 2.1.0
$ cat node_modules/serverless-localstack/package.json | grep version
  "version": "0.4.20"
$ cat serverless.yml | grep opt:stage
  stage: ${opt:stage, 'local'}
$ sls deploy
...
Serverless: Using serverless-localstack
...
```

**Comment by thetumper** [comment]
Resolved with 0.4.20!  Thanks much.
